cmd/build/helmify/static/README.md

```diff
@@ -70,6 +70,7 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | postInstall.labelNamespace.image.tag          | Image tag                                                                                                                                                                                                                                           | Current release version: `v3.9.0-beta.2`                                  |
 | postInstall.labelNamespace.image.pullPolicy   | Image pullPolicy                                                                                                                                                                                                                                    | `IfNotPresent`                                                            |
 | postInstall.labelNamespace.image.pullSecrets  | Image pullSecrets                                                                                                                                                                                                                                   | `[]`                                                                      |
+| postInstall.labelNamespace.extraRules         | Extra `Role` rules                                                                                                                                                                                                                                  | `[]`                                                                      |
```

Comment:
> nit: it looks like these are for the pre/post install hook roles only (not all roles), we should probably clarify that in the docs.
Ye, because there are 4 Service accounts (audit and controller use the same one, Namespace, CRD, Webhook)

Reply:
Right, I have no issues with what the code does, just want the documentation users see to reflect that reality.

Reply:
> Right, I have no issues with what the code does, just want the documentation users see to reflect that reality.

Any suggestions?
I can add extra rules for namespace service account role for example